README.md

```diff
@@ -1,3 +1,12 @@
+---
```

Comment:
There are tests (`markdown_markdown`).

I just happened to have added some metadata (celltoolbar) that shouldn't stay there. It seems like good practice indeed to be using the tool to maintain the README, so that we see things like this :)